.github/workflows/test.yml

```diff
@@ -73,6 +78,21 @@ jobs:
 
           skopeo copy oci-archive:dotnet-deps.tar docker-daemon:${{ env.runtime-deps-image-name }}
 
+          docker buildx build \
+            --platform=${buildx_platforms} \
+            --output type=oci,dest=dotnet-deps-sbom.tar \
+            -f dotnet-deps/Dockerfile.${{ matrix.ubuntu-release }} \
+            --target sbom-prep \
+            dotnet-deps
+
+          for arch in $archs; do
+            skopeo copy --override-arch "$arch" oci-archive:dotnet-deps-sbom.tar oci-archive:"dotnet-deps-sbom.$arch.tar"
+            syft packages \
+              -o spdx-json \
+              --name "${{ env.runtime-deps-image-name }}" \
+              oci-archive:"dotnet-deps-sbom.$arch.tar" > "dotnet-deps-${{ matrix.ubuntu-release }}-oci-$arch-root.sbom.spdx"
```

Comment:
`dotnet-deps-${{ matrix.ubuntu-release }}-oci-$arch-root.sbom.spdx` isn't enough. it should also have the .net version otherwise you won't be able to tell the difference between the sbom for .net6 and 7 for a given Ubuntu release

Reply:
Updated it with `dotnet-deps-${{ env.dotnet-version}}-${{ matrix.ubuntu-release }}-oci-$arch-root.sbom.spdx`.